src/model.ts

```diff
@@ -92,6 +149,7 @@ interface StartGameMessage {
   connection?: WebSocket;
   playerId?: string;
   gameId?: string;
+  allotedTime?: number;
```

Comment:
Lets keep game field nomenclature consistent everywhere - ie. rename this to `timePerPlayer`